javascript/src/URDFClasses.js

```diff
@@ -152,19 +154,27 @@ class URDFJoint extends URDFBase {
 
         }
 
+        let didUpdate = false;
+
+        this.mimicJoints.forEach(joint => {
+
+            didUpdate |= joint.updateFromMimickedJoint(...values);
```

Comment:
Can we use `didUpdate = didUpdate || joint.updateFromMimickedJoint(...values);` here? `|=` is technically a bitwise operator and will return a number which has bitten me in the past.

Reply:
Aha, sorry. How do you feel about `||=`?

Reply:
> Aha, sorry. How do you feel about `||=`?

I've had issues building with the new logical assignment operator previously. It's possible my babel config needs some updating but in the interest of keeping things simple for myself for now lets stick with `didUpdate = didUpdate || joint.updateFromMimickedJoint(...values);`.